<!-- {about} --><a name="about"></a>
<p></p>
<p>
  (From Wikipedia) The king scallop (<em>Pecten maximus</em>) is an edible
  saltwater clam, a marine bivalve mollusc in the family Pectinidae.
  <em>Pecten maximus</em> occurs in the eastern Atlantic along the European
  coast from northern Norway, south to the Iberian peninsula, it has also been
  reported off West Africa, off the Macaronesian Islands. In Great Britain and
  Ireland it is distributed all round the coast but it is uncommon and localised
  on the eastern North Sea coast.
</p>
<p>
  Picture credit:
  <a href="https://commons.wikimedia.org/wiki/File:Bivalvia_outline_icon.png"
    >Wikimedia Commons</a
  >
</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<p></p>
<h2 id="assembly" style="margin-bottom: 5px">Assembly</h2>
<p></p>
<p>
  Transcriptome assembly generated by the MolluscDB team using publicly
  available data [1]:
</p>

<p></p>
<p>
  Study:
  <a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP067223"
    >SRP067223</a
  >
</p>
<p>
  Selected runs:
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101476">SRR3101476</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101475">SRR3101475</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101480">SRR3101480</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101477">SRR3101477</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101483">SRR3101483</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101479">SRR3101479</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101487">SRR3101487</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101482">SRR3101482</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101481">SRR3101481</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101474">SRR3101474</a>
</p>
<p></p>
<p>
  The quality of the dataset was first assessed with FastQC v0.11.5 [2]. Then,
  raw reads were quality and adapter trimmed with BBDuk v36.92 [3] (<font
    face="monospace"
    >bbduk ... ref=adapters.fa threads=16 ktrim=r k=23 mink=11 hammingdistance=1
    ftm=5 tpe tbo qtrim=rl trimq=5 minavgquality=10 minlength=25</font
  >). Finally, trimmed reads were <i>de novo</i> assembled with Trinity v2.4.0
  [4] (<font face="monospace"
    >Trinity --seqType fq --max_memory 128G --CPU 16 --left ... --right ...
    --SS_lib_type RF</font
  >).
</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h2 id="annotation" style="margin-bottom: 5px">Annotation</h2>
<p></p>
<p>
  Transcripts were annotated by the MolluscDB team with TransDecoder v3.0.1 [5].
  BLASTp v2.6.0+ [6] searches against UniRef90 release 2017_03 [7] (<font
    face="monospace"
    >blastp -query longest_orfs.pep -db uniref90.fasta -max_target_seqs 1
    -outfmt 6 -evalue 1e-5 -out blastp.outfmt6</font
  >), and hmmscan v3.1b2 [8] searches against Pfam v31.0 [9] (<font
    face="monospace"
    >hmmscan --cpu 16 --domtblout pfam.domtblout Pfam-A.hmm
    longest_orfs.pep</font
  >) were included in the TransDecoder pipeline to maximize sensitivity for
  capturing ORFs (<font face="monospace"
    >TransDecoder.LongOrfs -t Trinity.fasta -S; TransDecoder.Predict --cpu 16
    --single_best_orf -t Trinity.fasta --retain_pfam_hits pfam.domtblout
    --retain_blastp_hits blastp.outfmt6</font
  >).
</p>
<!-- {annotation} -->
<!-- {references} -->
<h2>References</h2>
<p>
  &emsp;1.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/27160853">
    Characterization of the mantle transcriptome in bivalves: Pecten maximus,
    Mytilus edulis and Crassostrea gigas</a
  >
</p>
<p>
  &emsp;2.
  <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">
    Babraham Bioinformatics - FastQC A Quality Control tool for High Throughput
    Sequence Data</a
  >
</p>
<p>
  &emsp;3.
  <a href="http://jgi.doe.gov/data-and-tools/bbtools/">
    BBTools software package</a
  >
</p>
<p>
  &emsp;4.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572440">
    Full-length transcriptome assembly from RNA-Seq data without a reference
    genome</a
  >
</p>
<p>
  &emsp;5.
  <a href="https://github.com/TransDecoder/TransDecoder/wiki">
    TransDecoder: Find Coding Regions Within Transcripts</a
  >
</p>
<p>
  &emsp;6.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/20003500">
    BLAST+: architecture and applications.</a
  >
</p>
<p>
  &emsp;7.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/25398609">
    UniRef clusters: a comprehensive and scalable alternative for improving
    sequence similarity searches</a
  >
</p>
<p>
  &emsp;8.
  <a href="http://hmmer.org/">
    HMMER: biosequence analysis using profile hidden Markov models</a
  >
</p>
<p>
  &emsp;9.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/26673716">
    The Pfam protein families database: towards a more sustainable future</a
  >
</p>
<!-- {references} -->
