<!-- {about} --><a name="about"></a>
  <p>
  (From MarLIN) The soft-shell clam, <i>Mya arenaria</i>, is a species of edible saltwater clam, a marine bivalve mollusc in the family Myidae. <i>M. arenaria </i> lives in burrows up to 50 cm deep in sand, mud, sandy mud, and sandy gravels from the mid shore to the shallow sublittoral, sometimes to a depth of 192 m. Often abundant on estuarine flats where it can survive at salinities as low as 4-5 psu. It can be found on the European coast from the White Sea to northern Norway, in the Baltic Sea and Wadden Sea to Portugal as well as the Black Sea. Reported from Labrador to Georgia in the W. Atlantic and from North Sound, Alaska to California in the E. Pacific.
  <p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p>Transcriptome assembly produced by the molluscdb team using public data [1]:</p>
  <p></p>
  Study: <a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP045796">SRP045796</a><br>
  Selected runs: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR1560361">SRR1560361</a>
  <p></p>
  <p>
  The quality of the dataset was first assessed with FastQC v0.11.5 [2]. Then, raw reads were quality and adapter trimmed with BBDuk v36.92 [3] (<font face="monospace">bbduk ... ref=adapters.fa threads=16 ktrim=r k=23 mink=11 hammingdistance=1 ftm=5 tpe tbo qtrim=rl trimq=5 minavgquality=10 minlength=25</font>). Finally, trimmed reads were <i>de novo</i> assembled with Trinity v2.4.0 [4] (<font face="monospace">Trinity --seqType fq --max_memory 128G --CPU 16 --left ... --right ...</font>).
  </p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>
  Transcripts were annotated by the molluscDB team with TransDecoder v3.0.1 [5]. BLASTp v2.6.0+ [6] searches against UniRef90 release 2017_03 (<font face="monospace">blastp -query longest_orfs.pep -db uniref90.fasta -max_target_seqs 1 -outfmt 6 -evalue 1e-5 -out blastp.outfmt6</font>), and hmmscan v3.1b2 [7] searches against Pfam v31.0 (<font face="monospace">hmmscan --cpu 16 --domtblout pfam.domtblout Pfam-A.hmm longest_orfs.pep</font>) were included in the TransDecoder pipeline to maximize sensitivity for capturing ORFs (<font face="monospace">TransDecoder.LongOrfs -t Trinity.fasta; TransDecoder.Predict --cpu 16 --single_best_orf -t Trinity.fasta --retain_pfam_hits pfam.domtblout --retain_blastp_hits blastp.outfmt6</font>).
  </p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <p>&emsp;1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25589608">A phylogenetic backbone for Bivalvia: an RNA-seq approach</a></p>
  <p>&emsp;2. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"> Babraham Bioinformatics - FastQC A Quality Control tool for High Throughput Sequence Data</a></p>
  <p>&emsp;3. <a href="http://jgi.doe.gov/data-and-tools/bbtools/"> BBTools software package</a></p>
  <p>&emsp;4. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572440"> Full-length transcriptome assembly from RNA-Seq data without a reference genome</a></p>
  <p>&emsp;5. <a href="https://github.com/TransDecoder/TransDecoder/wiki"> TransDecoder: Find Coding Regions Within Transcripts</a></p>
  <p>&emsp;6. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20003500"> BLAST+: architecture and applications</a></p>
  <p>&emsp;7. <a href="http://hmmer.org/"> HMMER: biosequence analysis using profile hidden Markov models</a></p>
<!-- {references} -->

