<!-- {about} --><a name="about"></a>
  <p><br>
  (From MarLIN) The blue mussel (<em>Mytilus edulis</em>), is a sspecies of edible marine bivalve mollusc in the family Mytilidae. <em>M. edulis</em> occurs from the high intertidal to the shallow subtidal attached by fibrous byssus threads to suitable substrata. It can be found on the rocky shores of open coasts attached to the rock surface and in crevices, and on rocks and piers in sheltered harbours and estuaries, often occurring as dense masses. Its distribution ranges from the White Sea, south to southern France in the N.E. Atlantic. In the W. Atlantic it extends from the Canadian Maritimes south to North Carolina. It occurs on the coasts of Chile, Argentina, the Falkland Islands and the Kerguelen Isles.
  <p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p>
  Transcriptome assembly produced by the molluscdb team using public data:
  <p></p>
  Study: <a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP067223">SRP067223</a><br>
  Selected runs: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101489">SRR3101489</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101492">SRR3101492</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101494">SRR3101494</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101495">SRR3101495</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101496">SRR3101496</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101497">SRR3101497</a>,
  <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR3101501">SRR3101501</a>
  <p></p>
  <p>
  The quality of the dataset was first assessed with FastQC v0.11.5 [1]. Then, raw reads were quality and adapter trimmed with BBDuk v36.92 [2] (<font face="monospace">bbduk ... ref=adapters.fa threads=16 ktrim=r k=23 mink=11 hammingdistance=1 ftm=5 tpe tbo qtrim=rl trimq=5 minavgquality=10 minlength=25</font>). Finally, trimmed reads were <i>de novo</i> assembled with Trinity v2.4.0 [3] (<font face="monospace">Trinity --seqType fq --max_memory 128G --CPU 16 --left ... --right ... --SS_lib_type RF</font>).
  </p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>
  Transcripts were annotated by the molluscDB team with TransDecoder v3.0.1 [4]. BLASTp v2.6.0+ [5] searches against UniRef90 release 2017_03 (<font face="monospace">blastp -query longest_orfs.pep -db uniref90.fasta -max_target_seqs 1 -outfmt 6 -evalue 1e-5 -out blastp.outfmt6</font>), and hmmscan v3.1b2 [6] searches against Pfam v31.0 (<font face="monospace">hmmscan --cpu 16 --domtblout pfam.domtblout Pfam-A.hmm longest_orfs.pep</font>) were included in the TransDecoder pipeline to maximize sensitivity for capturing ORFs (<font face="monospace">TransDecoder.LongOrfs -t Trinity.fasta -S; TransDecoder.Predict --cpu 16 --single_best_orf -t Trinity.fasta --retain_pfam_hits pfam.domtblout --retain_blastp_hits blastp.outfmt6</font>).
  </p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <p>&emsp;1. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"> Babraham Bioinformatics - FastQC A Quality Control tool for High Throughput Sequence Data</a></p>
  <p>&emsp;2. <a href="http://jgi.doe.gov/data-and-tools/bbtools/"> BBTools software package</a></p>
  <p>&emsp;3. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572440"> Full-length transcriptome assembly from RNA-Seq data without a reference genome</a></p>
  <p>&emsp;4. <a href="https://github.com/TransDecoder/TransDecoder/wiki"> TransDecoder: Find Coding Regions Within Transcripts</a></p>
  <p>&emsp;5. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20003500"> BLAST+: architecture and applications</a></p>
  <p>&emsp;6. <a href="http://hmmer.org/"> HMMER: biosequence analysis using profile hidden Markov models</a></p>
<!-- {references} -->