<!-- {about} --><a name="about"></a>
<p></p>
<p>
  From Wikipedia: <i>Modiolus philippinarum</i> is a species of "horse mussel",
  a marine bivalve mollusc in the family Mytilidae. This species is present in
  the Red Sea, Madagascar and Indo-Pacific, from eastern Africa, to eastern
  Indonesia; north to Japan and south to Queensland and Western Australia. It
  inhabits sublittoral muds, at depth of 0-40m.
</p>
<p>
  Picture credit:
  <a href="https://commons.wikimedia.org/wiki/File:Bivalvia_outline_icon.png"
    >Wikimedia Commons</a
  >
</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<p></p>
<h2 id="assembly" style="margin-bottom: 5px">Assembly</h2>
<p></p>
<p>
  Genome assembly imported from
  <a href="http://datadryad.org/resource/doi:10.5061/dryad.h9942"
    >Dryad Digital Repository</a
  >.
</p>
<p>
  From publication [1]: The shallow-water horse mussels
  <i>M. philippinarum</i> were collected from a sandy shore at Ting Kok, Hong
  Kong (22&deg; 46.989' N, 114&deg; 21.389' E) during low tide in April 2014.
  The mussels were transported to HKBU and maintained in a seawater aquarium
  until use. The species identity was confirmed based on a morphological
  description. DNA from the adductor muscle was extracted using the DNeasy Blood
  & Tissue Kit (Qiagen, Hilden, Germany). Six libraries with insert sizes
  ranging from 180 bp to 5 kb were constructed and sequenced by Illumina
  Hiseq2000 and Hiseq2500 platforms. Low-quality reads and
  sequencing-adaptor-contaminated reads were trimmed and error corrected. Reads
  of mate-pair libraries were de-duplicated and sorted. The genome was assembled
  using Platanus version 1.21. Our genome-assembly pipeline involved assembling
  of DNA sequences using different parameters, merging results from different
  assemblies, removing redundant scaffolds, further scaffolding using
  transcripts, filling the gaps and correcting the errors.
</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<p></p>
<h2 id="annotation" style="margin-bottom: 5px">Annotation</h2>
<p>
  Gene models imported from
  <a href="http://datadryad.org/resource/doi:10.5061/dryad.h9942"
    >Dryad Digital Repository</a
  >.
</p>
<p>
  From publication [1]: The genome was annotated using MAKER version 3.0. MAKER
  was run twice, in order to train the gene models and increase the
  gene-predication accuracy. The gene models were further filtered by running
  OrthoMCL with the protein models from six species of mollusks (<i
    >Crassostrea gigas</i
  >, <i>Pinctada fucata</i>, <i>Lottia gigantea</i>, <i>Aplysia californica</i>,
  <i>Biomphalaria glabrata</i>, and <i>Octopus bimaculoides</i>) and sequences
  that could be clustered with these molluscs were retained. Transcriptome reads
  were then mapped to the remaining 'orphan' genes, and only those supported by
  at least 10 parts per million (ppm) reads were retained.
</p>
<!-- {annotation} -->
<!-- {references} -->
<h2>References</h2>
<p>
  &emsp;1.
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/28812709"
    >Adaptation to deep-sea chemosynthetic environments as revealed by mussel
    genomes</a
  >
</p>
<!-- {references} -->
