<!-- {about} --><a name="about"></a>
  <p></p>
  <p>From Wikipedia: The Mediterranean mussel (<i>Mytilus galloprovincialis</i>) is a species of marine bivalve mollusc in the family Mytilidae. It is an invasive species in many parts of the world, and also an object of aquaculture.<p>
  <p>Picture credit: <a href="https://commons.wikimedia.org/wiki/File:Mytilus_galloprovincialis_shell.jpg">Wikimedia Commons</a></p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <p></p>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p></p>
  <p>Genome assembly imported from NCBI.</p>
  <p>From publication [1]: For sequencing, we extracted 4 &#956g of DNA from muscle tissue from a single mussel extracted from the Ria of Vigo, Spain. Using this DNA, three sequencing libraries with insert sizes of 180, 500 and 800 bp were constructed and sequenced at BGI (Beijing Genomics Institute-China). These libraries were sequenced with the Illumina HiSeq2000 high-throughput platform using paired-end sequencing (100-bp reads). To clean the initial set of reads, we filtered out raw reads if they fulfilled any of these conditions: a) >5% ambiguous bases (represented by the letter N); b) poly-A structures; c) > = 20 bases with low quality scores; d) adapter contamination: reads with more than 10 bp aligned to the adapter sequence (no more than 3-bp mismatch allowed); or e) small insert-size reads in which paired reads overlapped more than or equal to 10 bp (10% mismatch allowed). Finally, we assembled de novo the reads resulting from the quality filtering step using SOAPdenovo v1.05 with parameters -K 31 -d 1 -M 1 -F-R.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p></p>
  <p>Gene models imported from NCBI.</p> 
  <p>From publication [1]: We used MAKER2 to predict in silico the gene models in our assembly. Additional <i>M. galloprovincialis</i> experimental sequence data coming from two other different sources were used to improve the gene prediction with MAKER2. These data consisted of our in-house RNA-Seq de novo assemblies from 4 different tissues, and protein sequences from the NCBI database. We used these 2 different datasets to build a first ab initio gene prediction model using the program SNAP. Finally, MAKER2 was used with the resulting previous SNAP results to accomplish a refined, more reliable gene prediction.</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <p>&emsp;1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26977809">A First Insight into the Genome of the Filter-Feeder Mussel Mytilus galloprovincialis</a></p>
<!-- {references} -->
