<!-- {about} --><a name="about"></a>  
  <p></p>
  <p>From Wikipedia: <i>Bathymodiolus platifrons</i>, described by Hashimoto and Okutani in 1994, is a deep-sea mussel that is common in hydrothermal vents and methane seeps in the Western Pacific Ocean. <i>B. platifrons</i> harbours methane-oxidizing bacteria in its gill, which help to transfer methane into material and energy to help it to thrive in such environments.</p>
  <p>Picture <a href="https://www.nature.com/articles/s41559-017-0121#f1">credit</a></p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <p></p>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p></p>
  <p>Genome assembly imported from <a href="http://datadryad.org/resource/doi:10.5061/dryad.h9942">Dryad Digital Repository</a>.</p>
  <p>From publication [1]: Individuals of <i>B. platifrons</i> were collected using the manned submersible Jiaolong from a methane seep (22&deg; 06.921' N, 119&deg; 17.131' E, 1,122 m deep) in the South China Sea in July 2013. The mussels were kept in an enclosed sample chamber placed in the sample basket of the submersible, and it took approximately three hours from sampling to sample preservation. Once the samples were brought to the upper deck of the mothership, the adductor muscle of an individual was dissected, cut into small pieces and immediately fixed in RNAlater. The sample was then transported to Hong Kong Baptist University (HKBU) on dry ice and stored at -80 &deg;C until use. DNA was extracted using the CTAB method, and its quality and quantity was checked with standard agarose gel electrophoresis and a Qubit Fluorometer, respectively. Nine libraries with insert sizes ranging from 180 bp to 16 kb were constructed using different preparation methods and sequenced by Illumina Hiseq2000 and Hiseq2500 platforms. Low-quality reads and sequencing-adaptor-contaminated reads were trimmed and error corrected. Reads of mate-pair libraries were de-duplicated and sorted. The genome was assembled using Platanus version 1.21. Our genome-assembly pipeline involved assembling of DNA sequences using different parameters, merging results from different assemblies, removing redundant scaffolds, further scaffolding using transcripts, filling the gaps and correcting the errors.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <p></p>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>Gene models imported from <a href="http://datadryad.org/resource/doi:10.5061/dryad.h9942">Dryad Digital Repository</a>.</p>
  <p>From publication [1]: The genome was annotated using MAKER version 3.0. MAKER was run twice, in order to train the gene models and increase the gene-predication accuracy. The gene models were further filtered by running OrthoMCL with the protein models from six species of mollusks (<i>Crassostrea gigas</i>, <i>Pinctada fucata</i>, <i>Lottia gigantea</i>, <i>Aplysia californica</i>, <i>Biomphalaria glabrata</i>, and <i>Octopus bimaculoides</i>) and sequences that could be clustered with these molluscs were retained. Transcriptome reads were then mapped to the remaining 'orphan' genes, and only those supported by at least 10 parts per million (ppm) reads were retained. </p>
  <p>The following proteins could not be imported correctly and were removed from the original gff: Bpl_scaf_29237-0.17, Bpl_scaf_29237-0.16 and Bpl_scaf_29237-0.15.</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <p>&emsp;1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28812709">Adaptation to deep-sea chemosynthetic environments as revealed by mussel genomes</a></p>
<!-- {references} -->
