<!-- {about} --><a name="about"></a>  
  <p></p>
  <p>From Wikipedia: The ezo giant scallop, scientific name <i>Mizuhopecten yessoensis</i> or <i>Patinopecten yessoensis</i>, is a marine bivalve mollusc from the family Pectinidae. This species is found around the far eastern Asian coast, from China, Korea, Japan and Sakhalin, and possibly as far north as the Kamchatka Peninsula and the Aleutian Islands. It is aquafarmed in China, South Korea, Japan, and Russia.</p>
  <p>Picture credit: <a href="https://commons.wikimedia.org/wiki/File:Japan_sea_animal,_Ezo_giant_scallop_(Mizuhopecten_yessoensis)_(15152229484).jpg">Wikimedia Commons</a></p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
  <p></p>
  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>
  <p></p>
  <p>Genome assembly imported from NCBI, see <a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_002113885.1">ASM211388v2</a> for details.</p>
  <p>From publication [1]: A one-year-old male <i>P. yessoensis</i> from a selfing family created with a hermaphroditic individual was used for WGS sequencing and assembly. High-quality genomic DNA was extracted from the adductor muscle of this inbred male using the conventional phenol/chloroform extraction method. Short-insert (180 bp, 300 bp and 500 bp) paired-end libraries and large-insert (2 kb and 5 kb) mate-pair libraries were prepared using Illumina's DNA library preparation kits following standard protocols. The 10 kb and 16 kb mate-paired libraries were prepared following the Cre-lox recombination-based protocol. The libraries were subjected to the paired-end 100 bp/150 bp sequencing on the Illumina HiSeq2000 platform. A modified version of SOAPdenovo was developed for efficient genome assembly to reduce the problem of high genome heterozygosity.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p></p>
  <p>Gene models imported from NCBI.
  <p>From publication [1]: Three de novo gene prediction tools, Augustus (v2.7), GlimmerHMM (v3.02) and SNAP (2006-07-28), were used to predict genes in the repeat-masked genome sequences. For homology-based gene prediction, protein sequences from <i>Crassostrea gigas</i>, <i>Lottia gigantea</i>, <i>Helobdella robusta</i>, <i>Anopheles gambiae</i>, <i>Caenorhabditis elegans</i>, <i>Drosophila melanogaster</i>, <i>Homo sapiens</i> and <i>Strongylocentrotus purpuratus</i> were aligned to the <i>P. yessoensis</i> genome using tblastn (v2.2.26; E-value &le; 1e-5), then the homologous genome sequences were aligned against the matching proteins using GeneWise (v2.4.1) for accurate spliced alignments. The RNA-Seq reads from different developmental stages and adult tissues/organs were aligned to the <i>P. yessoensis</i> genome using Tophat (2.0.11), and Cufflinks (2.1.1) was used to produce assembled transcripts and predict transcript structures. Gene predictions from the de novo approach, homology-based approach and RNA-Seq-based evidence were merged to form a comprehensive consensus gene set using the software EVM.</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <p>&emsp;1. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28812685">Scallop genome provides insights into evolution of bilaterian karyotype and development</a></p>
<!-- {references} -->
